packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -140,12 +149,13 @@ export class FilamentViewer extends LitElement {
     console.log('Scenario:', scenario.name);
     console.log('Lighting:', lightingBaseName);
 
+    // TODO : replace this with destroyAsset() when it's released
```

Comment:
Yup.  My fault.  Unfortunately this means you'll need to upgrade to 1.8.1 as soon as we release it.

By the way, the `destroyAsset` method will not only destroy non-ECS objects (e.g. VertexBuffer and Texture), it will also destroy entities.  So, it will allow you to remove your existing destroyEntity loop.  You will, however, still need to explicitly remove those entities from the Scene.  We'll also be exposing a `removeEntities` method to make this easier.